### ISSUE TITLE
Search results page broken down

### ISSUE THREAD

**Issue by krzysztof-grzybek** [infokiller/google-search-navigator#78]
Plugin was working great till today. After search, results page is scrolled down to the very bottom, and "next page" link at the bottom is focused.

**Comment by afilp** [comment]
I can verify this, it is not working well now.

**Comment by infokiller** [comment]
How did you install the extension? from the Chrome Webstore?

**Comment by krzysztof-grzybek** [comment]
@infokiller  yes, from the Chrome Webstore

**Comment by afilp** [comment]
@infokiller I built the source code, but I do not see any `output` folder, where should I look for the compiled result?

```
E:\R\google-search-navigator>yarn build --production
yarn run v1.7.0
$ gulp --production
[13:07:50] Using gulpfile E:\R\google-search-navigator\gulpfile.js
[13:07:50] Starting 'default'...
[13:07:50] Finished 'default' after 282 ms
Done in 10.33s.

E:\R\google-search-navigator>dir
 Volume in drive E is HDD
 Volume Serial Number is 34AF-D727

 Directory of E:\R\google-search-navigator

21/09/2018  11:14 πμ    <DIR>          .
21/09/2018  11:14 πμ    <DIR>          ..
21/09/2018  11:13 πμ               111 .clang-format
21/09/2018  11:13 πμ               232 .editorconfig
21/09/2018  11:13 πμ                83 .gitignore
21/09/2018  11:13 πμ    <DIR>          assets
21/09/2018  11:13 πμ    <DIR>          docs
21/09/2018  11:13 πμ               317 gulpfile.js
21/09/2018  11:13 πμ             1.088 LICENSE
21/09/2018  11:15 πμ    <DIR>          node_modules
21/09/2018  11:13 πμ            84.082 package-lock.json
21/09/2018  11:13 πμ               654 package.json
21/09/2018  11:13 πμ             3.486 README.md
21/09/2018  11:15 πμ    <DIR>          src
21/09/2018  11:13 πμ    <DIR>          tools
21/09/2018  11:13 πμ            49.627 yarn.lock
               9 File(s)        139.680 bytes
               7 Dir(s)  97.536.000.000 bytes free
```

**Comment by infokiller** [comment]
@krzysztof-grzybek the extension has been partially removed from the store (see #60 - waiting for Google to approve it again) so it hasn't received an update in a while. I  also no longer use the version from the store, because you can no longer install it, only if you had it before it was removed you can use it from there. 
So the bottom line is that I have no way to reproduce any issue in the web store version :(
Can you please try to clone it and install the latest master version manually and let me know if that solves it?
See https://github.com/infokiller/google-search-navigator/issues/71#issuecomment-419034669 for instructions.

**Comment by infokiller** [comment]
@afilp 

TLDR: Windows builds are not supported yet, but you can try a modern version by downloading a release: https://github.com/infokiller/google-search-navigator/releases/latest

Long story:
The `yarn build` command you're using is invoking gulp behind the scenes with [the gulp file](https://github.com/infokiller/google-search-navigator/blob/master/gulpfile.js). You can see that it builds in the src directory, but that's only the first step in the build.
You later need to invoke `tools/make-chrome.sh`, but that's a bash script and it looks like you're building it from Windows, so it won'e work.
Unfortunately, I never tried to build it from Windows (I mainly use Linux) and it probably won't work because the build scripts are written in bash.

**Comment by afilp** [comment]
@infokiller Thanks. Now that Windows 10 has `bash` support, I manage to compile bash-related code. For some reason, the compile of master fails even in bash (windows though):

![image](https://user-images.githubusercontent.com/7850073/45915243-bbafe100-be5a-11e8-820b-f22351d45360.png)

I wonder if anyone else with Windows here has the same problem.

**Comment by krzysztof-grzybek** [comment]
@infokiller  Thank You very much for explanation. It's annoying that google doesn't support it natively, now it's double annoying that they review this extension so long :( However, it's works great from build.

I don't know if You are aware of this (I didn't find it in issues), but after load it manually, I have two manifest errors: 
```
Unrecognized manifest key '-ms-preload'.
Unrecognized manifest key 'applications'.

**Comment by afilp** [comment]
For anyone interested, I **did compile in Windows**, using `bash`:

This is what is needed:

![image](https://user-images.githubusercontent.com/7850073/46008138-94f4e300-c0c4-11e8-916a-44c228b4d76d.png)

Note: You need to have support for `bash`, see here how: 
https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/

**Comment by idbrii** [comment]
I had the same issue as OP (v0.2.9 from the Chrome web store): search jumped to bottom and could only navigate between the next/prev search result page buttons at the bottom of the page.

Disabling my extension and installing the latest release (v0.2.14) fixed the issue for me:
https://github.com/infokiller/web-search-navigator/releases

Thanks for the great extension. I hope it's approved soon!
